src/app/frontend/login/component.ts

```diff
@@ -36,7 +36,7 @@ enum LoginModes {
 })
 export class LoginComponent implements OnInit {
   loginModes = LoginModes;
-  selectedAuthenticationMode = LoginModes.Kubeconfig;
+  selectedAuthenticationMode = localStorage.selectedAuthenticationMode;
```

Comment:
Use cookie service and save it in a cookie. Try to avoid using things that are not injected through the constructor. It's harder to follow where they're coming from and it's also harder for the angular build to do a proper tree shaking and other optimizations.

Reply:
done.